doc/mdc-migration.md

```diff
@@ -0,0 +1,54 @@
+# MDC Android migration
```

Comment:
```suggestion
# Material Design Components Migration
```

Is this fine @sanchita ? 🤔

Reply:
Sure. If you will be reusing this as an acronym anywhere in this doc then you might wanted to do this:
"Material Design Components (MDC)"